test/ai/threads/messages/messages.test.ts

```diff
@@ -33,4 +33,58 @@ describe('Message', () => {
     expect(txResult.result).toBeDefined();
     expect(Object.keys(messages).length).toBe(2);
   }, 10000);
+
+  it('get: should get message', async () => {
+    const message = await ainft.ai.chat.threads.messages.get(
+      threadId,
+      messageId,
+      objectId,
+      'openai',
+      'assistants',
+      tokenId,
+    );
+
+    expect(message.id).toBe(messageId);
+    expect(message.thread_id).toBe(threadId);
+    expect(message.role).toBe('user');
+    expect(message.content).toEqual([{ type: 'text', text: 'hello' }]);
+    expect(message.metadata).toEqual({});
+    expect(message.created_at).toBe(0);
+  });
+
+  it('list: should get message list', async () => {
+    const messages = await ainft.ai.chat.threads.messages.list(
+      threadId,
+      objectId,
+      'openai',
+      'assistants',
+      tokenId,
+    );
+
+    expect(messages.length).toBe(2);
+  });
+
+  it('update: should update message', async () => {
+    const txResult = await ainft.ai.chat.threads.messages.update(
+      threadId,
+      messageId,
+      {
+        objectId,
+        tokenId,
+        provider: 'openai',
+        api: 'assistants',
+        metadata: { key: 'value' },
+      }
+    );
+
+    const message = await ainft.ain.db
+      .ref(`/apps/${appId}/tokens/${tokenId}/ai/${aiName}/history/${address}/threads/${threadId}/messages/${messageId}`)
```

Comment:
(minor) how about..
```
....ref(
`...`
).getValue();
```
if it doesn't work for readability, please feel free to ignore this comment.

Reply:
thanks!